client/Packages/com.beamable/Editor/Dotnet/DotnetService.cs

```diff
@@ -70,7 +70,7 @@ public async Promise<List<string>> Run(string command)
 					_process.StartInfo.CreateNoWindow = true;
 					_process.StartInfo.UseShellExecute = false;
 					_process.StartInfo.Environment.Add("DOTNET_CLI_UI_LANGUAGE", "en");
-					_process.StartInfo.Environment.Add("BEAM_PATH", Path.GetFullPath(BeamCliUtil.CLI_PATH.Replace(".dll", "")));
+					_process.StartInfo.Environment.Add("BEAM_PATH", Path.GetFullPath(BeamCliUtil.CLI_PATH));

```

Comment:
This is not right, need to dive into the use of the DotnetService.Run and fix it to use the correct `BEAM_PATH`